ten-simple-rules-dockerfiles.Rmd

```diff
@@ -408,25 +411,56 @@ LABEL maintainer='D. Nüst <daniel.nuest@uni-muenster.de>' \
 'https://nuest.github.io/ten-simple-rules-dockerfiles/ten-simple-rules-dockerfiles.pdf' \
   org.opencontainers.image.version='1.0.0'
 
-# This ARG instruction adds a build-time variable with the default value 'unspecified'
-# Set it during `docker build` as follows to insert the current date and time into the 
-# label 'org.opencontainers.image.created':
-#   --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
-ARG BUILD_DATE unspecified
-
 LABEL org.opencontainers.image.vendor='Ten Simple Institute, Uni of Rules' \
   org.opencontainers.image.description='Reproducible workflow image' \
-  org.opencontainers.image.created=$BUILD_DATE \
   org.opencontainers.image.licenses='Apache-2.0'
-  
+
 LABEL edu.science.data.group.project='Find out something (Grant #123456)' \
   edu.science.data.group.name='Data Science Lab' \
   author.orcid='0000-0002-1825-0097'
 ```
 
 \normalsize
 
-_Listing&nbsp;3_: Example labels.
+_Listing&nbsp;3_: Commonly used labels.
+
+## Define versions, parameters, and paths once
+
+The [`ENV` instruction](https://docs.docker.com/engine/reference/builder/#env) in a `Dockerfile` allows for defining environment variables.
+These variables persist inside the container and can be useful for example for (i) setting software versions or paths and reusing them across multiple instructions to avoid mistakes, (ii) specifying metadata intended to be discovered by installed libraries or software, or (ii) adding binaries to the path (`PATH`) or library path (`LD_LIBRARY_PATH`).
```

Comment:
I think the second (ii) point should be a (iii)

Reply:
Fixed.